css/templates/travel/elements/_object-cover.css

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2016 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * object-cover
+ *
+ * Set the scale-mode of an img (or amp-img) element to "cover" instead of the
+ * default "stretch". This allows arbrairy scaling of img elements without
+ * distortion.
+ */
+
+.travel-object-cover,
```

Comment:
move this to main page.css , its hard to maintain all these files with one selector

Reply:
done